lib/beacon_api/api_spec.ex

```diff
@@ -0,0 +1,13 @@
+defmodule BeaconApi.ApiSpec do
+  @moduledoc false
+  alias OpenApiSpex.OpenApi
+  @behaviour OpenApi
+
+  @ethspec "beacon-node-oapi.json"
```

Comment:
nice! But it would be nice to do like we do with spec tests, we have a file with the version (see https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/.spectest_version) and a operation in the makefile to download the current version from github releases

Reply:
Also, we should throw an [`@external_resource <file>` tag](https://hexdocs.pm/elixir/1.14.5/Module.html#module-external_resource) for good measure.